src/pages/subjects/maths/index.jsx

```diff
@@ -2,7 +2,7 @@ import React from "react"
 
 import Layout from "../../../components/layout"
 import SEO from "../../../components/seo"
-import { H1, H6, H3 } from "../../../components/EasyText"
+import { H1, H2, H3, P } from "../../../components/EasyText"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['H3' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4474002)